docs/OURSPHINX-README.md

```diff
@@ -0,0 +1,10 @@
+# Details of Kytos Sphinx Theme 
+
+Here you'll find some directions and tips about the Kytos Sphinx Theme customization, it  all can be done right after cloning or forking this repository.  
```

Comment:
```suggestion
Here you'll find some directions and tips about the Kytos Sphinx Theme customization. It can all be done right after cloning or forking this repository.  
```